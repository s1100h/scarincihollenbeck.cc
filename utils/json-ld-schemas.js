import { CURRENT_DOMAIN, SITE_TITLE, SITE_EMAIL } from 'utils/constants';

const siteDescription = `${SITE_TITLE} is an alternative to a National 250 law firm. With offices in New Jersey, New York City, and the District of Columbia, we serve the niche practice areas most often required by institutions, corporations, entities, and the people who own and control them.`;

export const articleSchema = (body) => ({
  '@context': 'https://schema.org',
  '@graph': [
    {
      '@type': 'Article',
      publisher: {
        '@type': 'LegalService',
        name: SITE_TITLE,
        description: siteDescription,
        url: CURRENT_DOMAIN,
        image: `${CURRENT_DOMAIN}/images/no-image-found-diamond-750x350.png`,
        priceRange: '$$$$',
        telephone: '201-896-4100',
        email: SITE_EMAIL,
        hasMap:
          'https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d5375.6334332077395!2d-74.19960021040134!3d40.87248074962538!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c2fe482a4faf77%3A0x17707b4c60656ad!2s150%20Clove%20Rd%2C%20Little%20Falls%2C%20NJ%2007424!5e0!3m2!1sen!2sus!4v1659974943790!5m2!1sen!2sus',
        address: {
          '@type': 'PostalAddress',
          addressLocality: 'Little Falls',
          addressRegion: 'NJ',
          postalCode: '07424',
          streetAddress: '150 Clove Road, 9th Floor',
        },
        geo: {
          '@type': 'GeoCoordinates',
          latitude: '40.871378894707746',
          longitude: '-74.19506817327908',
        },
        sameAs: [
          'https://www.facebook.com/ScarinciHollenbeck/',
          'https://www.linkedin.com/company/scarinci-hollenbeck-llc',
          'https://twitter.com/s_h_law',
        ],
        openingHours: 'Mo-Fr 8:30-18:00',
      },
      ...body,
    },
  ],
});

export const buildBusinessSchema = () => ({
  '@context': 'https://schema.org',
  '@graph': [
    {
      '@type': 'LegalService',
      name: SITE_TITLE,
      description: siteDescription,
      url: CURRENT_DOMAIN,
      image: `${CURRENT_DOMAIN}/images/no-image-found-diamond-750x350.png`,
      priceRange: '$$$$',
      telephone: '201-896-4100',
      email: SITE_EMAIL,
      hasMap:
        'https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d5375.6334332077395!2d-74.19960021040134!3d40.87248074962538!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c2fe482a4faf77%3A0x17707b4c60656ad!2s150%20Clove%20Rd%2C%20Little%20Falls%2C%20NJ%2007424!5e0!3m2!1sen!2sus!4v1659974943790!5m2!1sen!2sus',
      address: {
        '@type': 'PostalAddress',
        addressLocality: 'Little Falls',
        addressRegion: 'NJ',
        postalCode: '07424',
        streetAddress: '150 Clove Road, 9th Floor',
      },
      geo: {
        '@type': 'GeoCoordinates',
        latitude: '40.871378894707746',
        longitude: '-74.19506817327908',
      },
      sameAs: [
        'https://www.facebook.com/ScarinciHollenbeck/',
        'https://www.linkedin.com/company/scarinci-hollenbeck-llc',
        'https://twitter.com/s_h_law',
      ],
      openingHours: 'Mo-Fr 8:30-18:00',
    },
    {
      '@type': 'WebSite',
      '@id': 'https://scarincihollenbeck.com/#website',
      url: 'https://scarincihollenbeck.com/',
      name: SITE_TITLE,
      description: siteDescription,
      publisher: {
        '@id': `${CURRENT_DOMAIN}/#organization`,
      },
      inLanguage: 'en-US',
    },
  ],
});

export const buildPersonSchema = (persons) => ({
  '@context': 'https://schema.org',
  '@graph': persons,
});

export const buildLocationSchema = (location) => ({
  '@context': 'https://schema.org',
  '@graph': [
    {
      '@type': 'LocalBusiness',
      name: SITE_TITLE,
      url: CURRENT_DOMAIN,
      logo: `${CURRENT_DOMAIN}/images/no-image-found-diamond.png`,
      image: location.image,
      address: {
        '@type': 'PostalAddress',
        streetAddress: location.streetAddress,
        addressLocality: location.addressLocality,
        addressRegion: location.addressRegion,
        postalCode: location.postalCode,
        addressCountry: location.addressCountry,
        telephone: location.telephone,
      },
      openingHours: ['Mo-Fr 08:00-18:00'],
      hasMap: location.mapAddress,
      geo: {
        '@type': 'GeoCoordinates',
        latitude: parseFloat(location.latitude),
        longitude: parseFloat(location.longitude),
      },
      priceRange: '$$$$',
      sameAs: [
        'https://www.facebook.com/ScarinciHollenbeck/',
        'https://www.linkedin.com/company/scarinci-hollenbeck-llc',
        'https://twitter.com/s_h_law',
      ],
    },
  ],
});

export const buildAttorneyProfileSchema = (
  name,
  url,
  imageUrl,
  socialMediaLinks,
  jobTitle,
) => {
  const links = socialMediaLinks.length > 0
    ? socialMediaLinks.map((link) => link.url)
    : [
      'https://www.facebook.com/ScarinciHollenbeck/',
      'https://www.linkedin.com/company/scarinci-hollenbeck-llc',
    ];

  return {
    '@context': 'https://schema.org/',
    '@graph': [
      {
        '@type': 'Person',
        name,
        url,
        image: imageUrl,
        sameAs: links,
        jobTitle,
        worksFor: {
          '@type': 'Organization',
          name: SITE_TITLE,
        },
      },
    ],
  };
};

export const STANDARD_SCHEMA = `{"@context":"https://schema.org/","@type":"WebSite","name":"${SITE_TITLE}","url":"${CURRENT_DOMAIN}","potentialAction":{"@type":"SearchAction","target":"{search_term_string}","query-input":"required name=search_term_string"}}`;
