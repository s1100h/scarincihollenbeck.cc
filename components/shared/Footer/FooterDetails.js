import {
  DetailsContainer,
  LinkTitle,
  LinkList,
  FooterColumn,
} from 'styles/Footer.style';

import { SITE_PHONE, SITE_FAX, SITE_EMAIL } from 'utils/constants';
import { BsFillEnvelopeFill } from 'react-icons/bs';
import { MdLocalPhone } from 'react-icons/md';
import { IoPrintSharp } from 'react-icons/io5';
import SocialNetworks from './SocialNetworks';

export default function FooterDetails() {
  return (
    <DetailsContainer>
      <FooterColumn>
        <LinkTitle>Contact Information</LinkTitle>
        <LinkList>
          <li>
            <a className="phone-mail-footer" href={`tel:${SITE_PHONE}`}>
              <span className="icon">
                <MdLocalPhone />
              </span>
              <span className="link-text">{SITE_PHONE}</span>
            </a>
          </li>
          <li>
            <IoPrintSharp size={20} />
            <span>{SITE_FAX}</span>
          </li>
          <li>
            <a className="phone-mail-footer" href={`mailto:${SITE_EMAIL}`}>
              <span className="icon">
                <BsFillEnvelopeFill />
              </span>
              <span className="link-text">{SITE_EMAIL}</span>
            </a>
          </li>
        </LinkList>
      </FooterColumn>

      <SocialNetworks />
    </DetailsContainer>
  );
}
