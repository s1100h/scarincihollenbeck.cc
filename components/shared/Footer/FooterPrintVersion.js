import React from 'react';
import empty from 'is-empty';
import Image from 'next/image';
import { CURRENT_DOMAIN, SITE_PHONE } from '../../../utils/constants';
import {
  ContactsPrintVersion,
  ContactsWrapperPrintVersion,
  FooterPrintVersionContainer,
} from '../../../styles/attorney-page/AttorneyPrintPage.style';
import Logo from '../../organisms/Navbar/Logo';

const FooterPrintVersion = ({ locations }) => {
  const qrCodeProduction = '/images/qrCodeHome.png';
  return (
    <FooterPrintVersionContainer>
      <div>
        <div className="d-flex flex-column">
          <p className="advertising-title-print">ATTORNEY ADVERTISING</p>
          <p className="advertising-text-print">
            No Aspect of the advertisement on this website or any of its pages,
            including awards and accolades, has been approved by the Supreme
            Court of NJ or NY.
          </p>
        </div>
        <Logo whiteVariant />
      </div>

      <ContactsWrapperPrintVersion>
        <ContactsPrintVersion>
          <div className="d-flex flex-column align-items-end">
            <div className="location-print-list mb-1">
              {!empty(locations)
                && locations.map(({ name }) => (
                  <div key={name} className="location-print-item">
                    {name}
                  </div>
                ))}
            </div>
            <div className="location-print-list">
              <div className="location-print-item">
                {CURRENT_DOMAIN.replace(/^https?:\/\//, '')}
              </div>
              <div className="location-print-item">{SITE_PHONE}</div>
            </div>
          </div>
        </ContactsPrintVersion>
        <div>
          <Image
            src={qrCodeProduction}
            alt="the Bio page"
            width={140}
            height={120}
          />
        </div>
      </ContactsWrapperPrintVersion>
    </FooterPrintVersionContainer>
  );
};

export default FooterPrintVersion;
