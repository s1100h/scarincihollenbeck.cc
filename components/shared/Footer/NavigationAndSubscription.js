import Link from 'next/link';
import { ContainerDefault } from 'styles/Containers.style';
import { ButtonRed } from 'styles/Buttons.style';
import MailingListIcon from 'components/common/icons/MailingListIcon';
import {
  FooterNavigation,
  NavHolder,
  NavWrapper,
} from '../../../styles/Footer.style';
import { footerNavList } from '../../../utils/constants';
import SubscriptionModal from '../../molecules/subscription/SubscriptionModal';

const NavigationAndSubscription = () => (
  <NavWrapper className="d-print-none" data-testid="footer-nav">
    <ContainerDefault>
      <NavHolder>
        <FooterNavigation>
          <ul>
            {footerNavList.map(({ linkTitle, link, id }) => (
              <li key={id}>
                <Link href={link}>{linkTitle}</Link>
              </li>
            ))}
          </ul>
        </FooterNavigation>
        <SubscriptionModal customClass="footer-subscription-btn">
          <ButtonRed as="span">
            <MailingListIcon />
            Join our mailing list
          </ButtonRed>
        </SubscriptionModal>
      </NavHolder>
    </ContainerDefault>
  </NavWrapper>
);

export default NavigationAndSubscription;
