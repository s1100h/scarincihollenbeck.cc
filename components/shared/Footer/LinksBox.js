import Link from 'next/link';
import { LinkTitle, LinkList, FooterColumn } from 'styles/Footer.style';
import empty from 'is-empty';

const linkTemplate = (is_target_blank, slugLink, label) => {
  if (is_target_blank) {
    return (
      <a href={slugLink} target="_blank" rel="noreferrer">
        {label}
      </a>
    );
  }

  return <Link href={slugLink}>{label}</Link>;
};

export default function LinksBox({
  linksArr,
  title,
  isTargetBlank,
  classList,
}) {
  return (
    <FooterColumn>
      {title && <LinkTitle>{title}</LinkTitle>}
      <LinkList className={!empty(classList) ? classList : ''}>
        {linksArr.map((link) => (
          <li key={link?.id || link?.databaseId}>
            {linkTemplate(
              isTargetBlank,
              link?.slug ? link?.slug : link?.url || link?.uri,
              link?.label || link?.title,
            )}
          </li>
        ))}
      </LinkList>
    </FooterColumn>
  );
}
