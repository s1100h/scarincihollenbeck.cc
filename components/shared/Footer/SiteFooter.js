import CookieConsentMessage from 'components/shared/CookieConsentMessage';
import {
  FIRM_PAGES,
  OFFICE_LOCATIONS,
  CORE_PRACTICES,
  excludedPrintFooter,
} from 'utils/constants';
import {
  Advertising,
  BottomLinks,
  FooterDoubleColumn,
  FooterHolder,
  FooterWrapper,
  LinksSEOBox,
} from 'styles/Footer.style';
import { useRouter } from 'next/router';
import Image from 'next/image';
import Link from 'next/link';
import { ContainerDefault } from 'styles/Containers.style';
import { useGetIndustriesQuery } from '../../../redux/services/project-api';
import LinksBox from './LinksBox';
import FooterDetails from './FooterDetails';
import {
  cannabisLawColors,
  globalColor,
} from '../../../styles/global_styles/Global.styles';
import { getSlugFromUrl } from '../../../utils/helpers';
import NavigationAndSubscription from './NavigationAndSubscription';
import { FooterContent } from '../../../styles/Footer.style';
import SHDiamond from '../../../public/images/sh-mini-diamond-PNG.svg';

const sanitizeIndustries = (industries) => {
  if (!industries) return [];

  return industries.map((industry) => ({
    id: industry?.databaseId,
    label: industry?.title,
    url: industry?.uri,
  }));
};

const setFooterBackgroundColor = (page) => {
  const footerColorsMap = {
    cannabis: cannabisLawColors.cannabisColorDarkGray,
    'entertainment-and-media': globalColor.black,
  };
  return footerColorsMap[page];
};

export default function Footer() {
  const currentYear = new Date().getFullYear();
  const { pathname } = useRouter();
  const slug = getSlugFromUrl(pathname);
  const backgroundFooterColor = setFooterBackgroundColor(slug);
  const conditionForPrintFooter = excludedPrintFooter.includes(pathname);
  const { data: industries } = useGetIndustriesQuery();

  return (
    <FooterWrapper
      $backgroundFooterColor={backgroundFooterColor}
      data-testid="footer"
      $isPrint={conditionForPrintFooter}
    >
      <NavigationAndSubscription />

      <ContainerDefault>
        <FooterHolder>
          <FooterContent>
            <LinksSEOBox>
              <LinksBox title="Core Practices" linksArr={CORE_PRACTICES} />
              <LinksBox
                title="Industries"
                linksArr={sanitizeIndustries(industries?.data)}
              />

              <FooterDoubleColumn>
                <LinksBox title="Firm Pages" linksArr={FIRM_PAGES} />
                <LinksBox
                  title="Office Locations"
                  linksArr={OFFICE_LOCATIONS}
                />
              </FooterDoubleColumn>
            </LinksSEOBox>

            <FooterDetails />
          </FooterContent>

          <Advertising>
            <Image
              src={SHDiamond}
              alt="sh diamond logo favicon"
              width={110}
              height={100}
            />
            <section>
              <p className="advertising-title">ATTORNEY ADVERTISING</p>
              <p className="advertising-text">
                No Aspect of the advertisement on this website or any of its
                pages, including awards and accolades, has been approved by the
                Supreme Court of NJ or NY.
              </p>
              <p className="advertising-text">
                Prior results do not guarantee a similar outcome. @
                {currentYear}
                , Scarinci Hollenbeck, LLC, all rights reserved
              </p>
              <BottomLinks className="d-print-none">
                <li>
                  <Link href="/privacy-policy">Privacy policy</Link>
                </li>
                <li className="list-inline-item">
                  <Link href="/terms-of-use">Terms of use</Link>
                </li>
              </BottomLinks>
            </section>
          </Advertising>
        </FooterHolder>

        <CookieConsentMessage />
      </ContainerDefault>
    </FooterWrapper>
  );
}
