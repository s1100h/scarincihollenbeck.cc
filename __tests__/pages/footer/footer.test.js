import {
  fireEvent,
  render,
  screen,
  waitFor,
  within,
  act,
} from '@testing-library/react';
import FooterDetails from 'components/shared/Footer/FooterDetails';
import Footer from 'components/shared/Footer/SiteFooter';
import ReduxProvider from 'hoks/reduxTestHoc';
import { useRouter } from 'next/router';
import decodeResponse from 'utils/decodeResponse';
import renderer from 'react-test-renderer';
import { CORE_PRACTICES } from 'utils/constants';
import LinksBox from 'components/shared/Footer/LinksBox';
import SHDiamond from '../../../public/images/sh-mini-diamond-PNG.svg';

jest.mock('next/router', () => ({
  useRouter: jest.fn(),
}));

useRouter.mockReturnValue({
  pathname: '/',
  route: '/',
  query: {},
  asPath: '/',
});

jest.mock('utils/decodeResponse', () => ({
  __esModule: true,
  default: jest.fn(),
}));

const mockCategoriesData = [
  { id: 608, name: 'Business Law' },
  { id: 98, name: 'Firm News' },
  { id: 5734, name: 'Technology' },
  { id: 2124, name: 'Tax' },
  { id: 99, name: 'Firm Events' },
];

beforeAll(() => {
  global.fetch = jest.fn(() => Promise.resolve());
  decodeResponse.mockResolvedValue({ data: mockCategoriesData });
});

afterAll(() => {
  jest.resetAllMocks();
});

describe('Footer', () => {
  it('Footer panel', async () => {
    await act(async () => {
      render(
        <ReduxProvider>
          <Footer />
        </ReduxProvider>,
      );
    });

    const footer = screen.getByTestId('footer');
    expect(footer).toBeInTheDocument();

    const footerPanel = screen.getByTestId('footer-nav');
    expect(footerPanel).toBeInTheDocument();

    const subscribePopupOpener = within(footerPanel).getByRole('button', {
      name: /Join our mailing list/i,
    });
    expect(subscribePopupOpener).toBeInTheDocument();

    fireEvent.click(subscribePopupOpener);

    await waitFor(() => {
      const modal = footerPanel.querySelector('.modal-open');
      expect(modal).toBeInTheDocument();
    });

    const logos = within(footer).getAllByRole('img', {
      src: SHDiamond,
    });

    const footerLogo = logos.find(
      (button) => button.getAttribute('alt') === 'sh diamond logo favicon',
    );

    expect(footerLogo).toBeInTheDocument();

    const privacyLink = within(footer).getByRole('link', {
      name: 'Privacy policy',
    });
    const termsLink = within(footer).getByRole('link', {
      name: 'Terms of use',
    });

    expect(privacyLink).toHaveAttribute('href', '/privacy-policy');
    expect(termsLink).toHaveAttribute('href', '/terms-of-use');
  });

  it('Footer links', () => {
    const tree = renderer.create(<FooterDetails />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('Footer categories', () => {
    render(<LinksBox title="Core Practices" linksArr={CORE_PRACTICES} />);

    const listTitle = screen.getByText('Core Practices');
    expect(listTitle).toBeInTheDocument();

    const listItems = screen.getAllByRole('listitem');
    expect(listItems).toHaveLength(CORE_PRACTICES?.length);
  });
});
