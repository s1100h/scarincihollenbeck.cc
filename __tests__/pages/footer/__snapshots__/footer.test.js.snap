// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer links 1`] = `
<section
  className="sc-3ba57081-3 fdfIOl"
>
  <div
    className="sc-3ba57081-8 eNtoMt"
  >
    <p
      className="sc-3ba57081-9 bOpqeW"
    >
      Contact Information
    </p>
    <ul
      className="sc-3ba57081-10 ivDYlE"
    >
      <li>
        <a
          className="phone-mail-footer"
          href="tel:201-896-4100"
        >
          <span
            className="icon"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
              />
            </svg>
          </span>
          <span
            className="link-text"
          >
            201-896-4100
          </span>
        </a>
      </li>
      <li>
        <svg
          fill="currentColor"
          height={20}
          stroke="currentColor"
          strokeWidth="0"
          style={
            {
              "color": undefined,
            }
          }
          viewBox="0 0 512 512"
          width={20}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M400 96V56a8 8 0 00-8-8H120a8 8 0 00-8 8v40"
          />
          <rect
            fill="none"
            height="160"
            rx="4"
            ry="4"
            width="208"
            x="152"
            y="264"
          />
          <rect
            fill="none"
            height="160"
            rx="4"
            ry="4"
            width="208"
            x="152"
            y="264"
          />
          <path
            d="M408 112H104a56 56 0 00-56 56v208a8 8 0 008 8h56v72a8 8 0 008 8h272a8 8 0 008-8v-72h56a8 8 0 008-8V168a56 56 0 00-56-56zm-48 308a4 4 0 01-4 4H156a4 4 0 01-4-4V268a4 4 0 014-4h200a4 4 0 014 4zm34-212.08a24 24 0 1122-22 24 24 0 01-22 22z"
          />
        </svg>
        201-896-8660
      </li>
      <li>
        <a
          className="phone-mail-footer"
          href="mailto:info@sh-law.com"
        >
          <span
            className="icon"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555ZM0 4.697v7.104l5.803-3.558L0 4.697ZM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757Zm3.436-.586L16 11.801V4.697l-5.803 3.546Z"
              />
            </svg>
          </span>
          <span
            className="link-text"
          >
            info@sh-law.com
          </span>
        </a>
      </li>
    </ul>
  </div>
  <div
    className="sc-3ba57081-8 eNtoMt"
  >
    <p
      className="sc-3ba57081-9 bOpqeW"
    >
      Social Media
    </p>
    <ul
      className="sc-3ba57081-10 sc-3ba57081-11 ivDYlE fCktIG"
    >
      <li>
        <a
          href="https://www.linkedin.com/company/scarinci-hollenbeck-llc/"
          rel="noreferrer"
          target="_blank"
        >
          <span
            className="icon"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              />
            </svg>
          </span>
          LinkedIn
        </a>
      </li>
      <li>
        <a
          href="https://www.facebook.com/ScarinciHollenbeck"
          rel="noreferrer"
          target="_blank"
        >
          <span
            className="icon"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M400 32H48A48 48 0 0 0 0 80v352a48 48 0 0 0 48 48h137.25V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.27c-30.81 0-40.42 19.12-40.42 38.73V256h68.78l-11 71.69h-57.78V480H400a48 48 0 0 0 48-48V80a48 48 0 0 0-48-48z"
              />
            </svg>
          </span>
          Facebook
        </a>
      </li>
      <li>
        <a
          href="https://twitter.com/S_H_Law"
          rel="noreferrer"
          target="_blank"
        >
          <span
            className="icon"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M64 32C28.7 32 0 60.7 0 96V416c0 35.3 28.7 64 64 64H384c35.3 0 64-28.7 64-64V96c0-35.3-28.7-64-64-64H64zm297.1 84L257.3 234.6 379.4 396H283.8L209 298.1 123.3 396H75.8l111-126.9L69.7 116h98l67.7 89.5L313.6 116h47.5zM323.3 367.6L153.4 142.9H125.1L296.9 367.6h26.3z"
              />
            </svg>
          </span>
          X
        </a>
      </li>
    </ul>
  </div>
</section>
`;
